app/src/main/java/com/futurice/freesound/network/api/model/Sound.kt

```diff
@@ -28,51 +28,51 @@ import java.util.*
 @Fieldable
 data class Sound(
 
-        val id: Long?,
+        val id: Long,
         // The URI for this sound on the Freesound website.
-        val url: String?,
+        val url: String,
         // The name user gave to the sound.
-        val name: String?,
+        val name: String,
         // An array of tags the user gave to the sound.
-        val tags: List<String>?,
+        val tags: List<String>,
         // The description the user gave to the sound.
-        val description: String?,
+        val description: String,
         // Latitude and longitude of the geotag separated by spaces
         // (e.g. “41.0082325664 28.9731252193”, only for sounds that have been geotagged).
-        val geotag: GeoLocation?,
+        val geotag: String?,
         // The username of the uploader of the sound.
-        val username: String?,
+        val username: String,
         // Thumbnail image URLs of the waveform/spectral plot
         val images: Image,
         // Preview sounds URLs
-        val previews: Preview?,
+        val previews: Preview,
         // Duration in seconds
-        val duration: Float?,
+        val duration: Float,
         val created: Date) : KParcelable {
     private constructor(parcel: Parcel) : this(
```

Comment:
Was there some reason why we are not using the Android extensions @Parcelize? Should available since 1.1.4 with:

```groovy
androidExtensions {
    experimental = true
}
```

Reply:
It was not stable enough in 1.1.4. But since 1.2 we can try it out

Reply:
But that would be in different PR

Reply:
Yep different PR. I've raised one - I like to live on the edge.